### PR TITLE
Update master annotation file (6/11)

### PR DESCRIPTION
**Pull request by komalsrathi**
Update master annotation file for downstream analyses and for manuscript.

### CODE REVIEW

**Comment by komalsrathi** [comment]
Requires TMB calculation to be run first, so converting this to draft for now.

File: analyses/master_annotation/results/master_annotation.tsv [hunk 1]
```diff
@@ -20,13 +20,13 @@ Sample	Sequencing_Experiment	Annotation	Diagnosis	Diagnosis_Type	Tumor_Location
...
 7316-2152	Methylation, Phospho-Proteomics, RNA-Seq, WGS, Whole Cell Proteomics	Treatment naive	High-grade glioma/astrocytoma (WHO grade III/IV)	Initial CNS Tumor	Midline	Midline	Male	4402	[0,15]	DMG, H3 K28, TP53	(DMG) Diffuse Midline Glioma	1.333900945238336
 7316-2176	Methylation, Phospho-Proteomics, RNA-Seq, WGS, Whole Cell Proteomics	Treatment naive	High-grade glioma/astrocytoma (WHO grade III/IV)	Initial CNS Tumor	Other/Multiple locations/NOS	Hemispheric	Female	6158	(15,26]	HGG, H3 wildtype	(HGG) High Grade Glioma (not otherwise specified)	0.13894801512899335
-7316-2241	Methylation, Phospho-Proteomics, RNA-Seq, WGS_tumor_only, Whole Cell Proteomics	Treatment naive	Glioblastoma (WHO grade IV)	Initial CNS Tumor	Cortical	Hemispheric	Female	2373	[0,15]	HGG, H3 wildtype, TP53	(HGG) High Grade Glioma (not otherwise specified)	14.45059357341531
+7316-2241	Methylation, Phospho-Proteomics, RNA-Seq, WGS_tumor_only, Whole Cell Proteomics	Treatment naive	Glioblastoma (WHO grade IV)	Initial CNS Tumor	Cortical	Hemispheric	Female	2373	[0,15]	HGG, H3 wildtype	(HGG) High Grade Glioma (not otherwise specified)	14.45059357341531
```
Comment (jharenza) [inline analyses/master_annotation/results/master_annotation.tsv:23]: is loss of TP53 expected in the subtype here?

File: analyses/master_annotation/results/master_annotation.tsv [hunk 2]
```diff
@@ -20,13 +20,13 @@ Sample	Sequencing_Experiment	Annotation	Diagnosis	Diagnosis_Type	Tumor_Location
...
 7316-2536	Methylation, Phospho-Proteomics, RNA-Seq, WGS_tumor_only, Whole Cell Proteomics	Post-mortem	Glioblastoma (WHO grade IV)	Initial CNS Tumor	Midline	Mixed	Female	4656	[0,15]	DMG, H3 K28	(DMG) Diffuse Midline Glioma	13.922591115925137
 7316-255	Methylation, Phospho-Proteomics, RNA-Seq, WGS_tumor_only, Whole Cell Proteomics	Treatment naive	Glioblastoma (WHO grade IV)	Initial CNS Tumor	Cortical	Hemispheric	Female	6120	(15,26]	HGG, H3 wildtype	(HGG) High Grade Glioma (not otherwise specified)	11.004682798216274
-7316-2594	Phospho-Proteomics, RNA-Seq, WGS, Whole Cell Proteomics	Treatment naive	Astrocytoma (WHO grade III/IV)	Initial CNS Tumor	Midline	Midline	Male	2869	[0,15]	DMG, H3 K28, TP53	(DMG) Diffuse Midline Glioma	5.280024574901748
+7316-2594	Phospho-Proteomics, RNA-Seq, WGS, Whole Cell Proteomics	Treatment naive	Astrocytoma (WHO grade III/IV)	Initial CNS Tumor	Midline	Midline	Male	2869	[0,15]	DMG, H3 K28	(DMG) Diffuse Midline Glioma	5.280024574901748
```
Comment (jharenza) [inline analyses/master_annotation/results/master_annotation.tsv:29]: similarly here

File: analyses/tp53_nf1_score/results/TP53_NF1_snv_alteration.tsv [hunk 3]
```diff
@@ -154,29 +127,7 @@ chr17	31327591	31327593	+	In_Frame_Del	BS_FAZBX8AS	NF1
...
-chr17	31234093	31402364	NA	Loss	BS_QJ58E1KY	NF1
-chr17	31147092	31479153	NA	Loss	BS_1WY6405N	NF1
-chr17	30982875	31104675	NA	Loss	BS_JZ40GV74	NF1
-chr17	31104675	31604055	NA	Loss	BS_JZ40GV74	NF1
-chr17	30488458	31753722	NA	Loss	BS_DFTX5Q9V	NF1
-chr17	26966160	45573735	NA	Loss	BS_3HXCGJ4J	NF1
-chr17	30396520	31754324	NA	Loss	BS_KYVZWYMS	NF1
-chr17	26966060	32996449	NA	Loss	BS_GCG2PKQA	NF1
-NA	NA	NA	NA	NA	BS_J4E9SW51	No_TP53_NF1_alt
+NA	NA	NA	NA	NA	BS_3NX3RBCX	No_TP53_NF1_alt
```
Comment (jharenza) [inline analyses/tp53_nf1_score/results/TP53_NF1_snv_alteration.tsv:130]: seems like this sample should have a TP53 deletion per the last release.

File: analyses/tp53_nf1_score/results/TP53_NF1_snv_alteration.tsv [hunk 4]
```diff
@@ -192,16 +143,19 @@ NA	NA	NA	NA	NA	BS_00TRPEQX	No_TP53_NF1_alt
 NA	NA	NA	NA	NA	BS_A0NBZQPH	No_TP53_NF1_alt
 NA	NA	NA	NA	NA	BS_QWS88QXE	No_TP53_NF1_alt
 NA	NA	NA	NA	NA	BS_V3X7YF5J	No_TP53_NF1_alt
-NA	NA	NA	NA	NA	BS_ABZ3BK38	No_TP53_NF1_alt
 NA	NA	NA	NA	NA	BS_NJFK43N3	No_TP53_NF1_alt
 NA	NA	NA	NA	NA	BS_KAD49R68	No_TP53_NF1_alt
 NA	NA	NA	NA	NA	BS_F064HMTP	No_TP53_NF1_alt
+NA	NA	NA	NA	NA	BS_WJB33V17	No_TP53_NF1_alt
```
Comment (jharenza) [inline analyses/tp53_nf1_score/results/TP53_NF1_snv_alteration.tsv:149]: think we need to check each of these which are different from last release

**Review by jharenza — CHANGES_REQUESTED** [review]
hey @komalsrathi I made some comments about subtypes that probably should not be changing TP53 status based on last release results. Can you have a look? Also, can you stack your PRs on the right base branch - this is showing 41 changes bc it is on master, but should be stacked on some other branch. You can just edit the title and change the base branch, thanks!

**Comment by komalsrathi** [comment]
> hey @komalsrathi I made some comments about subtypes that probably should not be changing TP53 status based on last release results. Can you have a look? Also, can you stack your PRs on the right base branch - this is showing 41 changes bc it is on master, but should be stacked on some other branch. You can just edit the title and change the base branch, thanks!

I don't understand. I created this PR after these PRs:

1) merge files 
2) NF1 score model 
3) HGG subtyping 
4) TMB calculation

And each PR was created off of the previous PR. Screenshots below that the previous commits were incorporated in this PR:

Screenshot 2024-07-15 at 9 24 46 AM

Screenshot 2024-07-15 at 9 24 57 AM

**Pull request merged**